File: js/1.js
const sumMultThreeOrFive = (num) => {
  let sum = 0;
  for (let i = 0; i < num; i++) {
    if (i % 3 === 0 || i % 5 === 0) {
      sum += i;
    }
  }
  return sum;
};

let ans = sumMultThreeOrFive(1000);

console.log(ans);


// O(1) space
// O(n) time

File: python/1.py
def multiples_of_3_and_5(limit):
    sum = 0
    for num in range(3, limit):
        if (num % 3 == 0 or num % 5 == 0):
            sum += num
    return sum

limit = 1000
answer = multiples_of_3_and_5(limit)
print(answer)

# O(1) space
# O(n) time
